_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,204 @@
+
+- the [visible pixels][visible] of the rendered image resource contain text and it is [essential][] that the text is rendered with that specific presentation; or
+- the [visible pixels][visible] of the rendered image resource do not contain text expressing anything in a [human language][]; or
+- the [visible pixels][visible] of the rendered image resource contain text and the text is not a significant part of the visual content in the image; or
+- the test target is [purely decorative][].
+
+## Assumptions
+
+- There is no mechanism to change the rendered text in the image resource. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might be satisfied.
+- The specific presentation of the text rendered in the image resource can be achieved through formatted text. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might be satisfied.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+This rule is designed specifically for [SC 1.4.5 Images of Text][sc1.4.5] which includes exceptions to the images it applies to that are not part of [SC 1.4.9 Images of Text (No Exception)][sc1.4.9]. Therefore, some images that are inapplicable for this rule can be applicable to [SC 1.4.9 Images of Text (No Exception)][sc1.4.9].
+
+- [Understanding Success Criterion 1.4.5: Images of Text][sc1.4.5]
+- [Understanding Success Criterion 1.4.9: Images of Text (No Exception)][sc1.4.9]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `img` element references an image resource that does not contain text.
+
+```html
+<img src="/test-assets/shared/fireworks.jpg" alt="fireworks going off behind the Eiffel tower at night" />
+```
+
+#### Passed Example 2
+
+This `input` element references an image resource that does not contain text.
+
+```html
+<input type="image" src="test-assets/shared/file.svg" alt="New file" />
+```
+
+#### Passed Example 3
+
+This `svg` element displays an image resource that does not contain text.
+
+```html
+<svg width="2in" height="3in" xmlns="http://www.w3.org/2000/svg">
+	<image x="20" y="20" width="200px" height="100px" href="/test-assets/shared/fireworks.jpg">
+		<title>Fireworks in Paris</title>
+	</image>
+</svg>
+```
+
+#### Passed Example 4
+
+This `object` element references an image resource where text is not the most significant content.
+
+```html
+<object data="/test-assets/0va7u6/times_square.jpg" title="Picture of Times Square, New York"></object>
+```
+
+#### Passed Example 5
+
+This `img` element references an image resource that contains text but where the presentation of the text is essential to convey the information.
+
+```html
+<p>The following image illustrates the use of cursive writing in the late nineteenth century.</p>
+<img src="/test-assets/0va7u6/letter.jpg" alt="Nineteenth century handwritten letter" />
```

Comment:
This is independent of this particular SC but I would argue that the text alternative is probably insufficient -- if the hand writing is essential then the text alternative probably needs to describe that hand writing a little more. Maybe show only one character/letter and describe the glyph in just a little more detail?

Reply:
Also, the text alternative should contain all the text visible in the image! If it is an entire letter then it may need some form of longer description. If you show, again, just one letter then it may be a little easier.

Reply:
Coming to think of it, I think this would be better as an entire word rather than a single letter.

Reply:
Done